package.json

```diff
@@ -115,7 +115,7 @@
     "typescript": "4.5.4",
     "url-polyfill": "1.1.12",
     "url-search-params-polyfill": "8.1.1",
-    "vanilla-framework": "^3.14.0",
+    "vanilla-framework": "4.1.0",
```

Comment:
wow aren't there any breaking changes that could be happening with this?

Reply:
We can't really rebrand bubbles while not being on V4. (I mean we can, but then we'd kind of have to do the work twice cause at some point all the custom styles I'd have to write to make it _look_ like it's on V4 would need to removed). But yes, regardless, this needs to be thoroughly QA'd. Relevant discussion [here](https://chat.canonical.com/canonical/pl/wi8kg469sjrp5r1utprbf4b9fo)